app/Console/Commands/NetworkData/ProcessNetworkData.php

```diff
@@ -54,11 +54,20 @@ public function __construct()
      */
     public function handle()
     {
+        $this->info('Getting network data from VATSIM.');
+
+        if ($this->networkData->failed() || ! $this->networkData->json()) {
+            $this->error('VATSIM feed unavailable.');
+            exit();
```

Comment:
Would it be simpler to just return?

Reply:
This was me trying to be super explicit as to the fact that we are killing the process there and then.